flow/connectors/clickhouse/normalize.go

```diff
@@ -474,19 +473,18 @@ func (c *ClickHouseConnector) NormalizeRecords(
 		c.logger.Warn("failed to get chunking parts, proceeding without chunking", slog.Any("error", err))
 		numParts = 1
 	}
+	numParts = max(numParts, 1)
 
 	// This is for cases where currently normalizing can take a looooong time
 	// there is no other indication of progress, so we log every 5 minutes.
 	periodicLogger := shared.Interval(ctx, 5*time.Minute, func() {
-		c.logger.Info("[clickhouse-cdc] inserting batch...",
-			slog.Int64("startBatchID", normBatchID),
+		c.logger.Warn("[clickhouse-cdc] inserting batches...",
+			slog.Int64("lastNormBatchID", lastNormBatchID),
```

Comment:
`startBatchID` is misleading here because we define startBatchID to be inclusive

```
	return model.NormalizeResponse{
		StartBatchID: normBatchID + 1,
		EndBatchID:   endBatchID,
	}, nil
```